src/Files.App/Files.App.csproj

```diff
@@ -55,12 +55,13 @@
         <Content Update="Assets\FilesOpenDialog\Files.App.Launcher.exe">
             <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
         </Content>
-        <Content Include="7z64.dll">
+        <Content Condition="'$(Platform)' == 'x64'" Include="7z64.dll">
             <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-        </Content><Content Include="7z.dll">
+        </Content>
+		<Content Condition="'$(Platform)' == 'x86'" Include="7z.dll">
```

Comment:
Can you fix the spacing?

Reply:
Sure, VisualStudio somehow didn't display that and it only showed up in git, fixed!